gateway/assets/img/icons/menu.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="24" height="24" viewBox="0 0 24 24"><path d="M3,6H21V8H3V6M3,11H21V13H3V11M3,16H21V18H3V16Z" /></svg>
```

Comment:
Where did you get `menu.svg` from? Do we have usage rights?

Reply:
Sorry @alexellis , should have pasted the link here, too. I got the icons from [here](https://github.com/Templarian/MaterialDesign). I though it was the official Material Design Icon but it seems it's a community driven one. It's under Apache License 2.0 so it's usable, but I'll replace it with the [official one](https://github.com/google/material-design-icons). (Which is also under Apache License 2.0)